app/project_type/git.py

```diff
@@ -19,6 +19,19 @@ class Git(ProjectType):
     DIRECTORY_PERMISSIONS = 0o700
 
     @staticmethod
+    def _replace_invalid_path_characters_from_netloc(netloc):
+        """
+        Replace the following characters in netloc.
+        ':' -> '-' (':' is not a valid character in path on Windows)
+
+        :param netloc: Network location
+        :type netloc: str
+        :return: Path after replacement
+        :rtype: str
+        """
+        return netloc.replace(':', '-')
```

Comment:
I see. I went back and forth on this one. I think it's still the best to preserve the old replace behavior.